_clients/k8s-operator.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: default
+title: OpenSearch Kubernetes Operator
+nav_order: 210
+---
+
+The OpenSearch Kubernetes (K8s) Operator is an open-source K8s operator that helps automate the deployment, provisioning, of OpenSearch and OpenSearch Dashboards in a containerized environment. The operator can manage multiple OpenSearch clusters which can be scaled up and down depending on your needs. 
+
+## Installation 
+
+There are two ways to get started with the Operator:
+
+- [Use a Helm chart](#use-a-helm-chartuse-a-helm)
+- [Use a local installation](#use-a-local-installation)
+
+### Use a Helm chart
+
+If you use Helm to manage your K8s cluster, you can use the OpenSearch Kubernetes Operator's CNCF project stored in Artifact Hub, a web-based application for finding, installing, and publishing CNCF packages. 
+
+To begin, log into your K8s cluster and add the Helm repo from [Artifact Hub](https://opster.github.io/opensearch-k8s-operator/). 
```

Comment:
If this is the first instance of "repo" on the page, spell out first time and then abbreviate in subsequent mentions. (per terms style guide)